### DISCUSSION TITLE
Coverage for CBL Mariner

### DISCUSSION THREAD

**Discussion by huornlmj** [ComplianceAsCode/content#12288] [General]
Hi.  Are there any plans to provide coverage for CBL Mariner (Azure linux) ? https://github.com/microsoft/azurelinux/tree/2.0

**Comment by Mab879** [comment]
Products are added by the community or by the organization behind the product. I have not heard any plans to add support for CBL Mariner. As always, pull requests are welcome to add the product.

**Comment by huornlmj** [comment]
To add more to this, there is now a CIS benchmark for Azure linux: https://www.cisecurity.org/benchmark/azure_linux (previously called Mariner).  Having an OVAL or SCAP profile would be wonderful for consumers.
